src/plugins/imagery/Imagery_View.js

```diff
@@ -1,8 +1,8 @@
-import ImageryViewLayout from './components/ImageryViewLayout.vue';
+import ImageryView from './components/ImageryView.vue';
```

Comment:
This could be imported as `ImageryViewComponent` to avoid the naming conflict

Reply:
Done